diff --git a/Makefile.linux b/Makefile.linux
index 439fb8b..3a4d469 100644
--- a/Makefile.linux
+++ b/Makefile.linux
@@ -4,13 +4,13 @@ CXX := g++
 # # uncomment to disable OpenGL functionality
 # NO_OPENGL := true
 
-CXXFLAGS := -Idependencies/include -I/usr/include/eigen3 -I/usr/include/superlu -std=c++14
+CXXFLAGS := -Idependencies/include -I/usr/include/eigen3 -I/usr/include/superlu -I/usr/include/suitesparse -std=c++14
 ifdef NO_OPENGL
 	CXXFLAGS := $(CXXFLAGS) -DNO_OPENGL
 endif
 CXXFLAGS_DEBUG := -Wall -g -fsanitize=address
 CXXFLAGS_RELEASE := -Wall -Ofast -Wreturn-type -fopenmp
-LDFLAGS := -Ldependencies/lib -lpng -lz -ltaucs -llapack -lblas \
+LDFLAGS := -Ldependencies/lib -lpng -lz -ltaucs -llapack -lblas -llapacke \
     -lboost_filesystem -lboost_system -lboost_thread -ljson -lgomp -lalglib \
     -lcholmod -lsuperlu -lumfpack
 ifndef NO_OPENGL
